package.json

```diff
@@ -35,7 +35,6 @@
     "preview": false,
     "qna": "https://github.com/aws/aws-toolkit-vscode/issues",
     "activationEvents": [
-        "onStartupFinished",
```

Comment:
for reference, motivation in https://github.com/aws/aws-toolkit-vscode/pull/1188 was to ensure that Toolkit features are available in e.g. `template.yaml` and code files even if AWS icon wasn't yet clicked.

Reply:
In that case, the "correct" way is to activate on certain file extensions. I ended up removing it just for testing purposes, though we should probably clean-up the activation events and ensure everything is correct. `onStartupFinished` ain't bad, but it's also not the best way of doing it.